### PR TITLE
add initial instructions

### PR DESCRIPTION
**Pull request by garlick**
I thought this would be fun.  It turns out there was zero work to do - just change a couple of lines in a python script somebody else wrote!

### CODE REVIEW

**Comment by chu11** [comment]
this puts my emojis to shame!

File: code.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+# create the switch, add a pullup, start it with not being pressed
+button = DigitalInOut(board.SWITCH)
+button.switch_to_input(pull=Pull.DOWN)
+button_state = False
+
+# create the captouch element and start it with not touched
+touch = touchio.TouchIn(board.TOUCH)
+touch_state = False
+
+# print a string on keypress
```
Comment (chu11) [inline code.py]: maybe just say "this is for gnome, edit this for other stuff" or something.   Edit: it now occurs to me, this comment maybe is from the example from the company, not your comment

**Review by chu11 — APPROVED** [review]
awesome!  i obviously didn't test, just looked for english errors.

**Comment by grondo** [comment]
Haha `ƒirmware` 😂

**Comment by grondo** [comment]
(sorry, accidentally hit close with comment when commenting on my phone)

File: code.py (hunk 1 above)
Comment (garlick) [reply code.py]: Good point, comment that is there is now incorrect!  I went with.
```python
# This enters an ƒ character in gnome
```

**Comment by garlick** [comment]
Thanks!  Merging.

**Pull request merged**